packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFWebViewHostApi.m

```diff
@@ -0,0 +1,176 @@
+                                                       FlutterError *_Nullable))completion {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (nullable NSNumber *)
+    getEstimatedProgressInstanceId:(nonnull NSNumber *)instanceId
+                             error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (nullable NSString *)getTitleInstanceId:(nonnull NSNumber *)instanceId
+                                    error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)goBackInstanceId:(nonnull NSNumber *)instanceId
+                   error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)goForwardInstanceId:(nonnull NSNumber *)instanceId
+                      error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)loadFileUrlInstanceId:(nonnull NSNumber *)instanceId
+                          url:(nonnull NSString *)url
+                readAccessUrl:(nonnull NSString *)readAccessUrl
+                        error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)loadFlutterAssetInstanceId:(nonnull NSNumber *)instanceId
+                               key:(nonnull NSString *)key
+                             error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)loadHtmlStringInstanceId:(nonnull NSNumber *)instanceId
+                          string:(nonnull NSString *)string
+                         baseUrl:(nullable NSString *)baseUrl
+                           error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)reloadInstanceId:(nonnull NSNumber *)instanceId
+                   error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)setAllowsBackForwardNavigationGesturesInstanceId:(nonnull NSNumber *)instanceId
```

Comment:
The setters are all awkward ordering-wise. `setBackForwardForWebViewWithIdentifier:isAllowed:error:`?

Reply:
I'd prefer to leave `NavigationGestures` there, I need a lot context to know what a `BackForward` mean. 
The name becomes very long.

Is navigationGesture not enough? Are there other navigation gestures besides backforward navigation gesture?If not maybe:
`setNavigationGesturesForWebViewWithIdentifier:isAllowed:error:`

Reply:
> I'd prefer to leave `NavigationGestures` there, I need a lot context to know what a `BackForward` mean.

Yes, I'm not sure why I dropped that. My name is clearly wrong since it doesn't disable actually going back or forward, just the gestures.

Having a long name on a method that's only ever going to be called by generated code (and tests) doesn't seem like aa big deal. That's definitely not the longest signature I've ever seen :)